.github/workflows/deploy.yml

```diff
@@ -71,7 +71,7 @@ jobs:
         run: |
           git config user.name github-actions
           git config user.email github-actions@github.com
-          git add -A .
+          git add -A . -- ':!*.json'
```

Comment:
```suggestion
          git add *.xml SpoilerSeasonEnabled
```

Reply:
I like not adding all here.

Are we in the `out` directory already or need to add that? 🤔

Reply:
read the line 3 above https://github.com/Cockatrice/Magic-Spoiler/pull/288/files#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3L70

Reply:
`working-directory` 👍